docs: add debug information (#160)
electron/notarize: README.md

If you use the new Notary Tool method you will need to set the `teamId` option. To get this ID, go to your [Apple Developer Account](https://developer.apple.com/account), then click on "Membership details", and there you will find your Team ID. This link should get you there directly: https://developer.apple.com/account#MembershipDetailsCard

## Debug

> Debug is used to display logs and messages, you can use `export DEBUG=electron-notarize:notarytool` when necessary.

Reviewer: ```suggestion
[`debug`](https://www.npmjs.com/package/debug) is used to display logs and messages. You can use `export DEBUG=electron-notarize*` to log additional debug information from this module.
```

If you use the new Notary Tool method you will need to set the `teamId` option. To get this ID, go to your [Apple Developer Account](https://developer.apple.com/account), then click on "Membership details", and there you will find your Team ID. This link should get you there directly: https://developer.apple.com/account#MembershipDetailsCard

> ## Debug
> Debug is used to display logs and messages; remember to export DEBUG=electron-notarize:notarytool when necessary.

Reviewer 2: ```suggestion
## Debug

Debug is used to display logs and messages, you can use `DEBUG=electron-notarize:notarytool` when necessary.
```
